### DISCUSSION TITLE
Rich markup inside of Markdown

### DISCUSSION THREAD

**Discussion by polyedre** [Textualize/rich#1249] [General] [answered]
It is possible to use [console markup](https://rich.readthedocs.io/en/stable/markup.html#rendering-markup) inside of `Table` and `Panel`:

```py
from rich import print
from rich.table import Table

print(Table("[bold cyan]test[/bold cyan]"))
```

However, it does not work with the `Markdown` object:

```
from rich import print
from rich.markdown import Markdown

print(Markdown("[bold cyan]test[/bold cyan]"))
```

Result:
```
[bold cyan]test[/bold cyan]                                                                                      
```

Is it a feature that could be added to rich ?

**Comment by willmcgugan** [comment, marked as answer]
Afraid not. It wouldn’t technically be markdown if there is another markup overlayed on top of it.

Have a look at markdown.py if you want to add more colour. The Markdown class was designed to be easy to customise.

**Reply by polyedre** [reply]
Ok thanks!
